src/Authentication/Services/SystemUserService.cs

```diff
@@ -49,8 +49,7 @@ public SystemUserService(ISystemUserRepository systemUserRepository, ISystemRegi
             {                
                 IntegrationTitle = request.IntegrationTitle,
                 ProductName = request.ProductName,
```

Comment:
I suggest that we use the same PR to change the model names. Remi had raised a concern about different names for the same data in different places. To be consistent, we will remove the word product from our models. We call it systemname, systemid and not productname/product id. Also are we referring the supplier to the systemvendor or apivendor?

Reply:
ok